hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketAllocator.java

```diff
@@ -269,8 +269,9 @@ public String toString() {
 
   // Default block size in hbase is 64K, so we choose more sizes near 64K, you'd better
   // reset it according to your cluster's block size distribution
+  // The real block size in hfile may a little larger than the size we configured ,
```

Comment:
The real block size in hfile **maybe** a little larger.....

Reply:
fixed,thanks for review.